File: tests/Fibonacci.pas
program Fibonacci;

var
   Fibonacci1 : integer;
   Fibonacci2 : integer;
   temp : integer;
   count : integer;

begin
   write('First ten Fibonacci numbers are: \n');
   count := 0;
   Fibonacci1 := 0;
   Fibonacci2 := 1;
   repeat
      write(Fibonacci2 div 7);
      temp := Fibonacci2;
      Fibonacci2 := Fibonacci1 + Fibonacci2;
      Fibonacci1 := Temp;
      count := count + 1;
   until count = 10;
   write('\n');

end.

File: tests/Fibonacci.c
#include <stdio.h> 
 

int main(void){
int fibonacci1;
int fibonacci2;
int temp;
int count;
printf("First ten Fibonacci numbers are: \n");
count = 0;
fibonacci1 = 0;
fibonacci2 = 1;
do { 
printf(fibonacci2 / 7);
temp = fibonacci2;
fibonacci2 = fibonacci1 + fibonacci2;
fibonacci1 = temp;
count = count + 1;
} while(!(count == 10));
printf("\n");
return 0; 
}